authors/apps/notifications/backends.py

```diff
@@ -0,0 +1,199 @@
+from .models import Notification
+from django.urls import reverse
+from authors.apps.core.models import email_dispatch
+import threading
+
+
+class NotificationDispatch:
+    def profile_followed(self, request, followed_profile):
+        """
+        notify users of new followers on their profiles
+
+        Args:
+            - request(Request): request object of the follower
+            - followed_profile(ProfileModel):
+
+        """
+        follower_profile = request.user.profile
+        if 'PROFILE_FOLLOW' not in followed_profile.notification_perms or \
```

Comment:
@malfahad for such scenarios, use a mapper such as a dictionary to prevent typos
```{ 'PROFILE_FOLLOW': 'PROFILE_FOLLOW' }
```